scripts/devnet/check.sh

```diff
@@ -38,3 +39,10 @@ while true; do
 
   sleep 1
 done
+
+# Check the offline RPC, which should be initialized at that point. It should be at the genesis height, so 0.
+height=$(get_sync_height ${FOREST_OFFLINE_RPC_PORT})
```

Comment:
There is chance offline server has not become healthy here. How about moving `check.sh` to a docker compose step with proper depends-on setups

Reply:
There is a 10 minute timeout on the retries. In my book that's enough, so I leave it up to you whether or not this goes into a separate docker container.

Reply:
It will be spun right after Lotus has initialised without any need for a sync, without waiting for connecting to Lotus, nor going through 5 different migrations over 20 epochs.

If it's not up and running at that point, something is definitely not right with the offline server itself.